ApollosProject/apollos-prototype
# makes markers tappable, campuses tappable, and always show current campus first

## DESCRIPTION

### What does this PR do, or why is it needed?

This PR handles a slew of issues / tweaks that we discovered as valuable during the QA process (and were mentioned by both willow and christ fellowship) 

The enhancements are: 

1. The campus cards themselves are now tappable
2. Tapping on a marker will change your current "focused" campus to that marker
3. If you have a current selected campus, it will always be the first campus in the list (and the one that you see when you open the mapview) 

### What design trade-offs/decisions were made?

This component should eventually move into its own package.

### How do I test this PR?

1. Open the Location Finder. 
2. Tap on a campus marker that's far from you (for example, Dallas for those of us in Cincy)  
3. Tap on the Campus Card (not the "Choose Campus" button)
4. Open Location Finder again. The campus you selected should be the first campus in the list.

### What automated tests did you write?

n/a

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
